docs/database/redis.md

```diff
@@ -10,7 +10,7 @@ The Semantic Conventions for [Redis](https://redis.com/) extend and override the
 that describe common database operations attributes in addition to the Semantic Conventions
 described on this page.
```

Comment:
Is this to say that attributes `database-spans.md` which do not appear on `redis.md` are not part of redis DB semantic conventions? If so (and I believe this is the case or else I'd ask why `db.collection.name` is omitted), then this is not immediately obvious based on the text.

Reply:
Good catch, let me follow up on this

Reply:
resolving and will follow up in a separate PR